gnc/sonar/include/BVWrapper.h

```diff
@@ -0,0 +1,48 @@
+#include <iostream>
+#include <iterator>
+
+#include <bvt_sdk.h>
+
+#include <opencv2/core/core.hpp> 
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <ros/package.h>
+
+namespace bv{
+	const int BW8 = 0;
```

Comment:
https://google.github.io/styleguide/cppguide.html#Spaces_vs._Tabs
I'd be okay with 4 spaces, but 2 is the standard